src/app-layout-toolbar/__tests__/app-layout-toolbar.test.tsx

```diff
@@ -35,7 +36,7 @@ describe('AppLayoutToolbar component', () => {
     );
   });
 
-  test('triggerless navigation', () => {
+  test('triggerless navigation', async () => {
```

Comment:
Here and in some cases below some tests have become async

This is needed because focus behavior is async now. I checked with a11y squad, the change is backward compatible